File: Java/03.02.2022/SmallestLargest.java
import java.util.*;
public class SmallestLargest {
    
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);

        System.out.println(("Enter number: "));
        String num = sc.nextLine();

        int[] arr = new int[num.length()];
        for(int i = 0; i < arr.length; i++){
            arr[i] = Integer.parseInt(Character.toString(num.charAt(i)));
        }

        int smallest = arr[0];
        int largest = arr[0];

        for(int i = 0; i < arr.length; i++){
            if(arr[i] < smallest)
                smallest = arr[i];
            if(arr[i] > largest)
                largest = arr[i];
        }

        System.out.println("Largest = " + largest);
        System.out.println("Smallest = " + smallest);
    }
}

File: DAA_LAB/Lab1_22.07.2022/smallest_largest.cpp
#include <iostream>
using namespace std;

int main()
{
    int n;
    cout << "Enter n" << endl;
    cin >> n;
    int arr[n];
    cout << " Enter values for array" << endl;
    for (int i = 0; i < n; i++)
    {
        cin >> arr[i];
    }

    int largest = 0;
    int smallest = 0;

    for (int i = 0; i < n; i++)
    {
        if (arr[i] > arr[largest])
            largest = i;

        if (arr[i] < arr[smallest])
            smallest = i;
    }

    cout << "Smallest = " << arr[smallest] << " largest = " << arr[largest] << endl;

    return 0;
}